workloads/router-perf-v2/common.sh

```diff
@@ -167,7 +169,7 @@ gen_mb_config(){
       "keep-alive-requests": '${keepalive_requests}',
       "clients": '${clients}'
     }'
-  done <<< $(oc get route -n http-scale-${termination} --no-headers | awk '{print $2}')
+  done < /tmp/temp-route-mb.txt
   echo "]") | python -m json.tool > http-scale-${termination}.json
```

Comment:
It would be easier to modify these lines to something like:

for host in $(oc get route -n http-scale-${termination} --no-headers | awk '{print $2}'); do


done